services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProviderConfig.java

```diff
@@ -55,9 +55,33 @@
     public static final String ENABLED_FROM_METADATA  = "enabledFromMetadata";
     public static final String AUTHN_CONTEXT_COMPARISON_TYPE = "authnContextComparisonType";
     public static final String AUTHN_CONTEXT_CLASS_REFS = "authnContextClassRefs";
-    public static final String AUTHN_CONTEXT_DECL_REFS = "authnContextDeclRefs";
-    public static final String SIGN_SP_METADATA = "signSpMetadata";
-
+    public static final String AUTHN_CONTEXT_DECL_REFS = "authnContextDeclRefs";   
+    public static final String SIGN_SP_METADATA = "signSpMetadata"; 
+    // entity attributes
+    public static final String RESEARCH_SCHOLARSHIP = "researchAndScholarship";
+    public static final String GEANT_DATA_PROTECTION_CODE = "geantDataProtectionCode";
+    // description extensions
+    public static final String REGISTRATION_AUTHORITY = "registrationAuthority";
+    public static final String REGISTRATION_POLICY = "registrationPolicy";
+    // UIinfo fields
+    public static final String DISPLAY_NAME = "displayName";
+    public static final String DESCRIPTION = "description";
+    public static final String INFORMATION_URL = "informationURL";
+    public static final String PRIVACY_STATEMENT_URL = "privacyStatementURL";
+    public static final String LOGO = "logo";
+    public static final String LOGO_HEIGHT = "logoHeight";
+    public static final String LOGO_WIDTH = "logoWidth";
+    // organization- contact person
+    public static final String ORGANIZATION_NAME = "organizationName";
+    public static final String ORGANIZATION_DISPLAY_NAME = "organizationDisplayName";
+    public static final String ORGANIZATION_URL = "organizationURL";
+    public static final String CONTACT_TYPE = "contactType";
+    public static final String CONTACT_COMPANY = "contactCompany";
+    public static final String CONTACT_GIVEN_NAME = "contactGivenName";
+    public static final String CONTACT_SURNAME = "contactSurname";
+    public static final String CONTACT_EMAIL_ADDRESS = "contactEmailAddress";
+    public static final String CONTACT_TELEMPHONE_NUMBER = "contactTelephoneNumber";
```

Comment:
Prefix all metadata constants by `MD_` / `MDUI_` to distinguish descriptive properties from those used for flow.

Reply:
ok. Do you want to change also to change constant value, getters and setters?
For example MD_DESCRIPTION = "mdDescription" ,ng-model="identityProvider.config.mdDescription", getMdDescription, setMdDescription.

Reply:
That would be great.

Reply:
I will do it.